System.IO.Abstractions/FileWrapper.cs

```diff
@@ -21,6 +27,18 @@ public override void AppendAllLines(string path, IEnumerable<string> contents, E
             File.AppendAllLines(path, contents, encoding);
         }
 
+#if NETCOREAPP2_0
+        public override async Task AppendAllLinesAsync(string path, IEnumerable<string> contents, CancellationToken cancellationToken)
```

Comment:
Here you can remove the `async`/`await` and return a `Task.CompletedTask` as well (below too).

Reply:
Gotcha, will update.

Reply:
Thanks!